lib/profileCache.js

```diff
@@ -90,7 +90,10 @@ module.exports = function profileCache(server, options) {
         scope: scope
       }}};
       return await server.methods.profileCache.get.cache.drop(req);
```

Comment:
@vladikoff this line is throwing a 500 error, am I missing something ?

Reply:
@deeptibaghel do you get this when running the tests?